src/operation/problem_results.jl

```diff
@@ -409,6 +409,13 @@ function read_variables(
     return Dict(encode_key_as_string(k) => v for (k, v) in result_values)
 end
 
+"""
+Return the values for all variables.
+"""
+function read_variables(res::IS.Results)
```

Comment:
Consistency concerns:
1. We have this function for ProblemResults. It accepts kwargs for parameters like start_time. We don't have it for the others. We could delete this set of functions.
```
function read_variables(res::ProblemResults; kwargs...)
```
2. We could add `kwargs...` to all the new functions and forward them. We would need to test all combinations. It could be messy. It's probably easier to say that these functions return all values for all variables. I would probably delete the functions mentioned in (1) but am not certain.

Reply:
I think is best to remove the methods and see what happens, some of those functions seem too generic and can be confusing.